modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/DeploymentServiceTestCase.java

```diff
@@ -170,6 +170,11 @@
   private static final File echoTestJarFile =
       new CompilerUtils.JarCompiler().compiling(getResourceFile("/org/foo/EchoTest.java")).compile("echo.jar");
 
+  private static final File defaulServiceSchedulerJarFile = new CompilerUtils.JarCompiler()
+      .compiling(getResourceFile("/org/mule/service/scheduler/MockSchedulerService.java"),
+                 getResourceFile("/org/mule/service/scheduler/MockSchedulerServiceProvider.java"))
+      .compile("mule-module-service-mock-scheduler-4.0-SNAPSHOT.jar");
```

Comment:
Is reuqired that the version is the same as the mule version? If not, use 1.0 or similar